exporters/etw/BUILD

```diff
@@ -0,0 +1,52 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
```

Comment:
Modify the Bazel build to include this only on Windows.

Reply:
@mishal23 - something along the lines described here:
https://blog.bazel.build/2019/02/11/configurable-builds-part-1.html

Reply:
We briefly discussed this with @jsuereth that you can for now keep this excluded the way you are excluding it with "-"

Reply:
Please open a bug/ticket to clean up the Bazel build to remove the need for this though.

Reply:
Sorry, I changed my mind here given the impact of day-to-day development using Bazel on non-windows.  See comment above.